src/System.Text.Encoding.CodePages/src/System.Text.Encoding.CodePages.csproj

```diff
@@ -30,6 +30,7 @@
     <Compile Include="System\Text\ISO2022Encoding.cs" />
     <Compile Include="System\Text\ISCIIEncoding.cs" />
     <Compile Include="System\Text\SBCSCodePageEncoding.cs" />
+    <Compile Include="System\Text\TestEncoderFallback.cs" />
```

Comment:
This should not be included here.  The src project is for the shipping assembly, not for tests.

Reply:
Removed.